yt_dlp/extractor/france24.py

```diff
@@ -0,0 +1,63 @@
+import json
+import re
+
+from .common import InfoExtractor
+
+
+class France24IE(InfoExtractor):
+    IE_NAME = 'france24'
+    _VALID_URL = r'https?://www\.france24\.com/'
```

Comment:
should make an effort to capture the slug as the `display_id`, and we don't want to match the index page of the site
```suggestion
    _VALID_URL = r'https?://www\.france24\.com/[a-z]{2}/(?:[^/?#]+/)+(?P<id>\d{8}-[\w-]+)'
```

Reply:
Actually, on the main page, you get the livestream (e.g. `https://www.france24.com/en/` or `https://www.france24.com/fr/`, including when redirected from `https://www.france24.com`).

Reply:
Is it a Youtube livestream?

Reply:
Yes.